NETProvider/src/FirebirdSql.Data.UnitTests/FbCommandTests.cs

```diff
@@ -730,6 +756,15 @@ public void NoCommandPlanTest()
 				Assert.DoesNotThrow(() => { plan=cmd.CommandPlan; });
 				Assert.IsEmpty(plan);
 			}
+			}
+			finally
+			{
+				using (var cmd = Connection.CreateCommand())
+				{
+					cmd.CommandText = "drop table NoCommandPlanTest";
```

Comment:
My idea was to drop all tables, procedures, triggers, ... and create these again. Fresh database. So whatever was done there is cleaned up. And you don't think about it. 

But maybe it's not that fast at the end. Your idea with cleaning up just the data might be faster.